python/lsst/pipe/analysis/analysis.py

```diff
@@ -1404,6 +1406,14 @@ def plotSkyObjects(self, skyObjCat, description, plotInfoDict, log, zpLabel=None
             skyFluxArr = flux[finiteFlux]
             skyFluxErrArr = fluxErr[finiteFlux]
             clippedStats = calcQuartileClippedStats(skyFluxArr, nSigmaToClip=5.0)
+            # add sigma-clipped mean to verify metrics tracker
+            if fluxStr == 'base_CircularApertureFlux_9_0_instFlux':
+                skyObjectMeanCalib = photoCalib.instFluxToNanojansky(clippedStats.mean) * units.nanojansky
+                skyObjectStdCalib = photoCalib.instFluxToNanojansky(clippedStats.stdDev) * units.nanojansky
```

Comment:
Spaces around multiplication operators removed, and operations moved to lines `1490` and `1493` to avoid line length limitations in their former locations.

Reply:
Following the removal of `photoCalib` (due to the fact that skyObjCat values are already in AB units), all mentions of `photoCalib` have been removed from these PRs. Consequently, these two lines have been re-jigged accordingly.